tmt/utils.py

```diff
@@ -1420,13 +1420,9 @@ def render_run_exception(exception: RunError) -> List[str]:
             line_summary = f"{min(len(output_lines), OUTPUT_LINES)}/{len(output_lines)}"
             output_lines = output_lines[-OUTPUT_LINES:]
 
-        lines += [
-            f'{name} ({line_summary} lines)',
-            OUTPUT_WIDTH * '~'
-            ] + output_lines + [
-            OUTPUT_WIDTH * '~',
-            ''
-            ]
+        lines += [f'{name} ({line_summary} lines)',
+                  OUTPUT_WIDTH * '~',
+                  *output_lines, OUTPUT_WIDTH * '~', '']
```

Comment:
Nitpicking, but: could you preserve the formatting? We're collecting a list of lines in `lines`, and `*output_lines, OUTPUT_WIDTH * `~`, ''` sort of hides it's actually N + 2 more lines by being on the same line of the code.